scripts/operations_tests/operations_tests.py

```diff
@@ -330,17 +346,11 @@ def main():
     parser.add_argument("-k", dest="match", type=str, help="only run tests which match the given substring expression")
     parser.add_argument("--gui", dest="gui", action="store_true", help="Show GUI comparison for differences")
 
-    global args
     args = parser.parse_args()
```

Comment:
I think the arg parsing in main is not needed any more, because it is in TestRunner